build/full/Dockerfile

```diff
@@ -51,42 +51,62 @@ RUN curl -L https://dlcdn.apache.org//jmeter/binaries/apache-jmeter-${JMETER_VER
 
 ENV PATH /opt/apache-jmeter-${JMETER_VERSION}/bin/:$PATH
 
-
 RUN curl -L https://github.com/flanksource/askgit/releases/download/v0.4.8-flanksource/askgit-linux-amd64.tar.gz -o askgit.tar.gz && \
-    tar xf askgit.tar.gz && \
-    mv askgit /usr/local/bin/askgit && \
-    rm askgit.tar.gz && \
-    wget http://mirrors.kernel.org/ubuntu/pool/main/o/openssl/openssl_1.1.1f-1ubuntu2.19_amd64.deb && \
-    dpkg -i openssl_1.1.1f-1ubuntu2.19_amd64.deb && \
-    rm openssl_1.1.1f-1ubuntu2.19_amd64.deb
-
+  tar xf askgit.tar.gz && \
+  mv askgit /usr/local/bin/askgit && \
+  rm askgit.tar.gz && \
+  wget http://mirrors.kernel.org/ubuntu/pool/main/o/openssl/openssl_1.1.1f-1ubuntu2.19_amd64.deb && \
+  dpkg -i openssl_1.1.1f-1ubuntu2.19_amd64.deb && \
+  rm openssl_1.1.1f-1ubuntu2.19_amd64.deb
+
+# The best developer experience for load testing
 ENV K6_VERSION=v0.44.0
 RUN curl -L https://github.com/grafana/k6/releases/download/${K6_VERSION}/k6-${K6_VERSION}-linux-amd64.tar.gz -o k6.tar.gz && \
-    tar xvf k6.tar.gz && \
-    mv k6-${K6_VERSION}-linux-amd64/k6 /usr/local/bin/k6 && \
-    rm k6.tar.gz
+  tar xvf k6.tar.gz && \
+  mv k6-${K6_VERSION}-linux-amd64/k6 /usr/local/bin/k6 && \
+  rm k6.tar.gz
 
+# Benthos is a high performance and resilient stream processor
 RUN curl -Lsf https://sh.benthos.dev | bash -s -- 4.15.0
 
+# Commandline tool for running SQL queries against JSON, CSV, Excel, Parquet, and more
 RUN curl -L https://github.com/multiprocessio/dsq/releases/download/v0.23.0/dsq-linux-x64-v0.23.0.zip -o dsq.zip && \
-    unzip dsq.zip && \
-    mv dsq /usr/local/bin/dsq && \
-    rm dsq.zip
-
-RUN curl -L https://github.com/stern/stern/releases/download/v1.25.0/stern_1.25.0_linux_amd64.tar.gz -o stern.tar.gz && \
-    tar xvf stern.tar.gz && \
-    mv stern /usr/local/bin/stern && \
-    rm stern.tar.gz
+  unzip dsq.zip && \
+  mv dsq /usr/local/bin/dsq && \
+  rm dsq.zip
 
 # install CA certificates
 COPY --from=builder /app/.bin/canary-checker /app
 
+# Install alexellis/arkade as root
+RUN curl -sLS https://get.arkade.dev | sh
+
+# Install Azure CLI (need to install as root)
+RUN curl -sL https://aka.ms/InstallAzureCLIDeb | bash
+
 RUN mkdir /opt/database
 RUN groupadd --gid 1000 canary
 RUN useradd canary --uid 1000 -g canary -m -d /var/lib/canary
 RUN chown -R 1000:1000 /opt/database
 RUN chown -R 1000:1000 /app
 USER canary:canary
 
+# Install CLIs with arkade as canary user
+RUN arkade get kubectl
+RUN arkade get stern
+RUN arkade get jq
+RUN arkade get yq
+ENV PATH="${PATH}:/var/lib/canary/.arkade/bin/"
+ENV PATH="${PATH}:/var/lib/canary/bin/"
+
+# Install AWS CLI
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
```

Comment:
Do this in 1 command and then remove the zip file to save space in the docker image - same applies to the others